react/ItemContext.tsx

```diff
@@ -27,3 +27,5 @@ export const useItemContext = () => {
 
   return context
 }
+
+export default { useItemContext }
```

Comment:
The `useItemContext` is already a named export (see the `export const useItemContext` just above). Why are you exporting it as the default export as well?

Reply:
because of VTEX IO imports. If you only export it as a named export, the import in other apps won't work

Reply:
Oh, right, got it!